content/docs/fundamentals/converting/resolving-errors@ja.md

```diff
@@ -0,0 +1,348 @@
+---
+$title: 検証エラーを解決する
+---
+
+[TOC]
+
+ここでは、AMP ページでの AMP 検証エラーを確認し、解決していきます。お使いのコンソールでは、異なる順序でエラーが表示される場合がありますので、ご了承ください。
+
+## 文字セットを指定する
+
+まず、次のエラーを修正します。
+
+<pre class="error-text">
+必須タグ「meta charset=utf-8」がないか、正しくありません。
+</pre>
+
+AMP でテキストを正しく表示するには、ページの文字セットを指定する必要があります。また、`<head>` タグの最初の子として、meta charset 情報を指定する必要があります。このタグを最初に指定する理由は、meta charset タグより前に追加されたコンテンツを再解釈するのを避けるためです。
+
+`<head>` タグの最初の行として次のコードを**追加**します。
+
+
+```html
+<meta charset="utf-8" />
+```
+
+ファイルを**保存**して、ページを再読み込みします。文字セットエラーが表示されなくなったことを確認します。
+
+## 正規リンクを指定する
+
+次のエラーを見てみましょう。
+
+<pre class="error-text">
+必須タグ「link rel=canonical」がないか、正しくありません。
```

Comment:
As console messages aren't translated, that is, they'll see it in English, this text should remain in English.